### ISSUE TITLE
Why are there no data returned after calling these two APIs, but there are actually advertising statistics in the background of this store?

### ISSUE THREAD

**Issue by a785817258** [allegro/allegro-api#9526]
Allegro Api：/sale/classified-seller-stats
Allegro request params：{"date.gte":"2024-07-29T06:12:26Z","date.lte":"2024-07-29T10:12:26Z"}
trace-id：27db84e6263d41ec
Allegro status:504 response：{"errors":[{"code":"ServiceTimeoutException","message":"An error occurred","details":null,"path":null,"userMessage":"Query response time exceeded. Try again."}]}
Allegro Api：/sale/classified-offers-stats
Allegro request params：{"offer.id":"15833897277"}
trace-id：6a6a74a22afc96e5
Allegro status:200 response：{"offerStats":[]}
![image](https://github.com/user-attachments/assets/a0317a90-1adc-4a55-8798-bc3d232d6578)

**Comment by a785817258** [comment]
@AureliuszBrussy

**Comment by PrzemyslawLukanowski** [comment]
These interfaces are not associated with each other. GET /sale/classified-seller-stats returns daily statistics collected for a list of **advertisements** (not offers).

**Comment by a785817258** [comment]
@PrzemyslawLukanowski  GET /sale/classified-seller-stats Why does this interface return 504?

**Comment by PrzemyslawLukanowski** [comment]
It seems there are issues on our side, I've forwarded this case to the team responsible for this service.

**Comment by a785817258** [comment]
@PrzemyslawLukanowski Is there any other API to get this data? For example, /billing/billing-entries?

**Comment by PrzemyslawLukanowski** [comment]
No, at this moment we do not provide such an endpoint. If anything changes in the future, we will inform about it on [our website](https://developer.allegro.pl/news).
